src/huggingface_hub/hf_file_system.py

```diff
@@ -287,7 +291,11 @@ def _ls_tree(
         resolved_path = self.resolve_path(path, revision=revision)
         path = resolved_path.unresolve()
         root_path = HfFileSystemResolvedPath(
-            resolved_path.repo_type, resolved_path.repo_id, resolved_path.revision, ""
+            resolved_path.repo_type,
+            resolved_path.repo_id,
+            resolved_path.revision,
+            "",
```

Comment:
(nit) I know it's not related to your PR but let's be explicit about what the empty string means

```suggestion
            path_in_repo="",
```